sal_interfaces/MTCamera/MTCamera_Telemetry.xml

```diff
@@ -1,948 +1,3407 @@
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>71</Count>
+        </item>
+        <item>
+            <EFDB_Name>temp7</EFDB_Name>
+            <Description>Board temperature 7</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>71</Count>
+        </item>
+        <item>
+            <EFDB_Name>temp8</EFDB_Name>
+            <Description>Board temperature 8</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>71</Count>
+        </item>
+        <item>
+            <EFDB_Name>temp9</EFDB_Name>
+            <Description>Board temperature 9</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>71</Count>
+        </item>
+    </SALTelemetry>
+    <SALTelemetry>
+        <Subsystem>MTCamera</Subsystem>
+        <EFDB_Topic>MTCamera_focal_plane_Ccd</EFDB_Topic>
+        <item>
+            <EFDB_Name>gDV</EFDB_Name>
+            <Description>GD 0 voltage</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Volt</Units>
+            <Count>197</Count>
+        </item>
+        <item>
+            <EFDB_Name>location</EFDB_Name>
+            <Description>Ccd Location</Description>
+            <IDL_Type>string</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>oDI</EFDB_Name>
+            <Description>OD 0 current</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>mA</Units>
+            <Count>197</Count>
+        </item>
+        <item>
+            <EFDB_Name>oDV</EFDB_Name>
+            <Description>OD 0 voltage</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Volt</Units>
+            <Count>197</Count>
+        </item>
+        <item>
+            <EFDB_Name>oGV</EFDB_Name>
+            <Description>OG 0 voltage</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Volt</Units>
+            <Count>197</Count>
+        </item>
+        <item>
+            <EFDB_Name>rDV</EFDB_Name>
+            <Description>RD 0 voltage</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Volt</Units>
+            <Count>197</Count>
+        </item>
+        <item>
+            <EFDB_Name>temp</EFDB_Name>
+            <Description>CCD temperature</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>197</Count>
+        </item>
+    </SALTelemetry>
+    <SALTelemetry>
+        <Subsystem>MTCamera</Subsystem>
+        <EFDB_Topic>MTCamera_focal_plane_Segment</EFDB_Topic>
+        <item>
+            <EFDB_Name>i</EFDB_Name>
+            <Description>Amplifier current</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>mA</Units>
+            <Count>3024</Count>
```

Comment:
Corner rafts aren't included?

Reply:
I think this is correct, the corner raft boards do not have the ability to measure amplifier currents.